### ISSUE TITLE
Login screen doesn't scroll & keyboard obscures login form

### ISSUE THREAD

**Issue by sderenSFDC** [forcedotcom/SalesforceMobileSDK-ReactNative#169]
By default, the login screen on android devices does not scroll when the keyboard appears. 

To fix this I had to make the following change in mobile_sdk/SalesforceMobileSDK-Android/libs/SalesforceSDK/AndroidManifest.xml:

```
<activity android:name="com.salesforce.androidsdk.ui.LoginActivity"
                    android:theme="@style/SalesforceSDK"
                    android:launchMode="singleInstance"
                    android:screenOrientation="portrait"
                    android:windowSoftInputMode="adjustResize" // <---- THIS
/> 
```

This is likely an Salesforce Android SDK-specific issue, but thought I would post here as I encountered this via the RN template app.  Is there a better way to fix this outside the SDK?

**Comment by wmathurin** [comment]
Thank you for your feedback. We will look into it and might upgrade our templates accordingly.

**Comment by wmathurin** [comment]
Fixed in dev - will be in our next release (8.2). See https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/2091
